### PR TITLE
test(integrations): Added telemetry for diffusers, trl, trlx, langchain, llama-index and stability-idk

### PR DESCRIPTION
**Pull request by ayulockin**
Fixes WB-NNNN
Fixes #NNNN

Description
-----------
The PR adds telemetry for diffusers, trl, trlx, langchain, llama-index and stability-idk.

Testing
-------
How was this PR tested?

Checklist
-------
- [ ] Include reference to internal ticket "Fixes WB-NNNN" and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Ensure PR title compliance with the [conventional commits standards](https://github.com/wandb/wandb/blob/main/CONTRIBUTING.md#conventional-commits)

### CODE REVIEW

**Review by dmitryduev — CHANGES_REQUESTED** [review]
Thanks @ayulockin!

We disabled all the import tests in nightly. They tend to frequently break due to third party library compatibility issues and people don't seem to have cycles to fix them. In any case, it seems that the only thing those tests actually test is telemetry and for that you can just mock out the libraries you are interested in, see: tests.pytest_tests.system_tests.test_core.test_telemetry_full.test_telemetry_imports

**Comment by raubitsj** [comment]
> Hey @dmitryduev thanks for the info.
> 
> If I understand clearly, I should then revert changes made to:
> 
> * .cicleci.config.yml
> * tests/functional_tests/t0_main/imports/12-batch12-requirements.txt
> * tests/functional_tests/t0_main/imports/12-batch12.py
> * tox.ini
> 
> Instead make changes to `tests.pytest_tests.system_tests.test_core.test_telemetry_full.test_telemetry_imports`

Revert all the test changes, no need to even make a pytest_tests.   just the telemetry proto buf changes if fine.

File: .circleci/config.yml [hunk 1]
```diff
@@ -1175,6 +1175,7 @@ workflows:
                 - "imports10"
                 - "imports11"
                 - "imports12"
+                - "imports13"
```
Comment (raubitsj) [inline .circleci/config.yml:1178]:
```suggestion
```

File: tests/functional_tests/t0_main/imports/12-batch12-requirements.txt [hunk 2]
```diff
@@ -1,3 +1,6 @@
 keras-cv
 pycocotools
 mmengine
+diffusers
+trl
+git+https://github.com/CarperAI/trlx
```
Comment (raubitsj) [inline tests/functional_tests/t0_main/imports/12-batch12-requirements.txt:6]:
```suggestion
```

File: tests/functional_tests/t0_main/imports/12-batch12.py [hunk 3]
```diff
@@ -25,10 +25,22 @@
...
     - 82  # mmengine
+  - :op:contains:
+    - :wandb:runs[0][telemetry][1] # imports init
+    - 83 # diffusers
+  - :op:contains:
+    - :wandb:runs[0][telemetry][1] # imports init
+    - 84 # trl
+  - :op:contains:
+    - :wandb:runs[0][telemetry][1] # imports init
+    - 85 # trlx
```
Comment (raubitsj) [inline tests/functional_tests/t0_main/imports/12-batch12.py:36]:
```suggestion
```

File: tests/functional_tests/t0_main/imports/12-batch12.py [hunk 4]
```diff
@@ -25,10 +25,22 @@
...
 """
 
+import diffusers  # noqa: F401
```
Comment (raubitsj) [inline tests/functional_tests/t0_main/imports/12-batch12.py:39]:
```suggestion
```

File: tests/functional_tests/t0_main/imports/12-batch12.py [hunk 5]
```diff
@@ -25,10 +25,22 @@
...
 import keras_cv  # noqa: F401
 import mmengine  # noqa: F401
+import trl  # noqa: F401
+import trlx  # noqa: F401
```
Comment (raubitsj) [inline tests/functional_tests/t0_main/imports/12-batch12.py:43]:
```suggestion
```

File: tox.ini [hunk 6]
```diff
@@ -9,7 +9,7 @@ envlist=
     docstrings,
     py{36,37,38,39,launch},
     func-s_{base,sklearn,metaflow,tf115,tf21,tf24,tf25,tf26,ray112,ray2,service,docs,
-            imports{1,2,3,4,5,6,7,8,9,10,11,12},noml,grpc}-py37,
+            imports{1,2,3,4,5,6,7,8,9,10,11,12,13},noml,grpc}-py37,
```
Comment (raubitsj) [inline tox.ini:12]:
```suggestion
            imports{1,2,3,4,5,6,7,8,9,10,11,12},noml,grpc}-py37,
```

File: tox.ini [hunk 7]
```diff
@@ -429,7 +429,7 @@ commands =
     cp .coverage coverage.xml cover-results/
     coverage report -m --ignore-errors --skip-covered --omit "wandb/vendor/*"
 
-[testenv:func-s_{base,sklearn,metaflow,tf115,tf21,tf24,tf25,tf26,ray112,ray2,service,py310,docs,imports1,imports2,imports3,imports4,imports5,imports6,imports7,imports8,imports9,imports10,imports11,imports12,noml,grpc,kfp}-{py36,py37,py38,py39,py310}]
+[testenv:func-s_{base,sklearn,metaflow,tf115,tf21,tf24,tf25,tf26,ray112,ray2,service,py310,docs,imports1,imports2,imports3,imports4,imports5,imports6,imports7,imports8,imports9,imports10,imports11,imports12,import13,noml,grpc,kfp}-{py36,py37,py38,py39,py310}]
```
Comment (raubitsj) [inline tox.ini:432]:
```suggestion
[testenv:func-s_{base,sklearn,metaflow,tf115,tf21,tf24,tf25,tf26,ray112,ray2,service,py310,docs,imports1,imports2,imports3,imports4,imports5,imports6,imports7,imports8,imports9,imports10,imports11,imports12,noml,grpc,kfp}-{py36,py37,py38,py39,py310}]
```

File: tox.ini [hunk 8]
```diff
@@ -478,6 +478,7 @@ commands =
     func-s_imports10-py{36,37,38,39,310}: yea {env:CI_PYTEST_SPLIT_ARGS:} --strict --shard imports10 run {posargs:--all}
     func-s_imports11-py{36,37,38,39,310}: yea {env:CI_PYTEST_SPLIT_ARGS:} --strict --shard imports11 run {posargs:--all}
     func-s_imports12-py{36,37,38,39,310}: yea {env:CI_PYTEST_SPLIT_ARGS:} --strict --shard imports12 run {posargs:--all}
+    func-s_imports13-py{36,37,38,39,310}: yea {env:CI_PYTEST_SPLIT_ARGS:} --strict --shard imports13 run {posargs:--all}
```
Comment (raubitsj) [inline tox.ini:481]:
```suggestion
```

**Review by raubitsj — APPROVED** [review]
Looks good

**Comment by ayulockin** [comment]
> Revert all the test changes, no need to even make a pytest_tests. just the telemetry proto buf changes if fine.

Thanks @raubitsj this simplify things for adding new libraries to track.

**Pull request merged**